docs/getting-started-guides/fedora/fedora_ansible_config.md

```diff
@@ -58,7 +59,7 @@ We will set the kube_ip_addr to '10.254.0.[1-3]', for now.  The reason we do thi
 [etcd]
 192.168.121.205
 
-[minions]
+[nodes]
```

Comment:
Yeah, I doubt this can be changed without also updating https://github.com/eparis/kubernetes-ansible